src/settings/ConditionsSettings.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import _ from 'lodash';
+
+import { Settings } from '@folio/stripes/smart-components';
+
+import Conditions from './patronBlocks/Conditions/Conditions';
+
+class ConditionsSettings extends Component {
+  static manifest = Object.freeze({
+    patronBlockConditions: {
+      type: 'okapi',
+      records: 'patronBlockConditions',
+      path: 'patron-block-conditions',
+    },
+  });
+
+  static propTypes = {
+    resources: PropTypes.shape({
+      patronBlockConditions: PropTypes.shape({
+        records: PropTypes.arrayOf(PropTypes.object).isRequired,
+      }).isRequired,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      patronBlockConditions: PropTypes.shape({
+        GET: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
+  };
+
+  getConditions = () => {
+    const {
+      resources: {
+        patronBlockConditions: {
+          records: patronBlockConditions,
+        }
+      }
+    } = this.props;
+    const routes = [];
+
+    patronBlockConditions.forEach((patronBlockCondition) => {
+      const {
+        id,
+        name,
+      } = patronBlockCondition;
+
+      function tempConditions() {
+        return <Conditions id={id} />;
+      }
+
+      routes.push({
+        route: id,
+        label: name,
+        component: tempConditions,
+      });
+    });
+
+    return routes;
+  }
+
+  shouldRenderSettings = () => {
+    const {
+      resources,
+    } = this.props;
+    const patronBlockConditions = _.get(resources, ['patronBlockConditions', 'records'], []);
```

Comment:
I know we have this pattern everywhere but we can now also use chaining operator. Could you replace this with:

`resources?.patronBlockConditions?.records ?? []`

You should be able to remove the lodash import after that.